### PR TITLE
Fix the `eex` compiled flaky test

### PR DESCRIPTION
**Pull request by scottming**
The previous CI was fine, and I rebased the latest branch, but the merge still fails the test

### CODE REVIEW

**Review by zachallaun — COMMENTED** [review]
This doesn't make sense to me in a `setup` block unless these compiler options are being changed in individual tests.

I feel like this should be in a `setup_all` and we should be resetting whatever the previous compiler options were on exit, since this is shared global state:

```elixir
setup_all do
  prev_compiler_options = Code.compiler_options()
  Code.compiler_options(parser_options: [columns: true, token_metadata: true])

  on_exit(fn ->
    Code.compiler_options(prev_compiler_options)
  end)

  :ok
end
```

File: apps/remote_control/test/lexical/remote_control/build/document/compilers/eex_test.exs [hunk 1]
```diff
@@ -26,6 +26,17 @@ defmodule Lexical.RemoteControl.Build.Document.Compilers.EExTest do
     Document.new("file:///file.eex", content, 0)
   end
 
+  setup_all do
+    prev_compiler_options = Code.compiler_options()
+    Code.compiler_options(parser_options: [columns: true, token_metadata: true])
```
Comment (scohen) [inline apps/remote_control/test/lexical/remote_control/build/document/compilers/eex_test.exs:31]: should we just call `Build.State.set_compiler_options()` ?

File: apps/remote_control/test/lexical/remote_control/build/document/compilers/heex_test.exs [hunk 2]
```diff
@@ -22,8 +22,15 @@ defmodule Lexical.RemoteControl.Build.Document.Compilers.HeexTest do
     Document.new("file:///file.heex", content, 0)
   end
 
-  setup do
+  setup_all do
+    prev_compiler_options = Code.compiler_options()
```
Comment (scohen) [inline apps/remote_control/test/lexical/remote_control/build/document/compilers/heex_test.exs:26]: same comment

File: apps/remote_control/test/lexical/remote_control/build/document/compilers/eex_test.exs (hunk 1 above)
Comment (scottming) [reply apps/remote_control/test/lexical/remote_control/build/document/compilers/eex_test.exs]: yeah, that's better.

**Review by scohen — APPROVED** [review]

**Pull request merged**